release-notes/Cube/Cube_Feature_Release_10.5/Cube_10.5.5.md

```diff
@@ -12,49 +12,130 @@ uid: Cube_Feature_Release_10.5.5
+The most prominent new EPM integration feature is the Topology app. If your system contains at least one correctly configured EPM Manager element (as detailed below), this app becomes available via a button in the Cube sidebar. Clicking the button will open a pane where you can select the front-end EPM Manager (in case more than one is available), select the topology chain, drill down to any of the topology levels in that chain, and open a card representing the data of the selected item. When you navigate between (docked) EPM cards, the filter selection will be updated to match the displayed card.
 
-> [!NOTE]
-> This new *Enable tracking to avoid duplicate inform acknowledgments (ACKs)* option is not selected by default and is not compatible with the existing *Resend all active alarms every:* option. It is also not compatible with the *Resend...* command, which can be selected after right-clicking an SNMP manager in the *SNMP forwarding* section of System Center.
+![Topology app](~/user-guide/images/EPMIntegration_Topology_app.png)
 
-#### EPM functionality is now fully integrated in DataMiner Cube [ID 42221]
+To make sure the Topology app is available, the following configuration is needed:
 
-<!-- MR 10.4.0 [CU14] / 10.5.0 [CU2] - FR 10.5.5 -->
+1. Make sure the DMS contains at least one front-end EPM Manager element.
+
+   For DataMiner Cube to detect these elements, they should be active, the *type* attribute of the *Display* tag in their connector should be set to "element manager", and they should contain a parameter named *ElementManagerType* set to 1. Also, their name should preferably start with "FE".
 
-All EPM functionality is now fully integrated in the UI of DataMiner Cube. Up to now, this required the *CPEIntegration* soft-launch option to be enabled.
+   Each front-end EPM Manager element will describe a topology chain and its associated filters.
 
-In the sidebar, you can now open the *Topology* pane, which allows you to select a topology chain, drill down to any of the topology level in that chain. On top of that, all EPM-related data can now also be accessed via the Surveyor or the Alarm Console.
+1. Go to *System Center* > *System settings* > *EPM config*, and add the front-end EPM Manager elements to the list.
 
-##### Configuration
+   In the Topology app, you will now be able to select the front-end managers and the corresponding topology chains.
 
-1. Make sure the DMS contains at least one EPM frontend manager element.
+##### EPM masking info<!-- 26002 -->
 
-   For DataMiner Cube to detect these elements, they should be active, the *type* attribute of the *Display* tag in their connector should be set to "element manager", and they should contain a parameter named *ElementManagerType* set to 1. Also, their name should preferably start with "FE".
+If an EPM object in the topology is masked, you can get information on who masked it and when via the *Masking info* option in the hamburger menu of the Topology app:
```

Comment:
I'm confused what is meant with "Topology app" in this context

Reply:
It's the name that was originally given to the new Topology pane in the EPM documentation, which has been in use already for some time. However, I'll clarify this in the release note so it's less confusing for users who aren't familiar with this yet.